src/lib/Libnet/net_server.c

```diff
@@ -485,6 +517,7 @@ engage_authentication(conn_t *pconn)
  *	routine associated with the socket is invoked.
  *
  * @param[in] waittime - Timeout for tpp_em_wait (poll)
+ * @param[in] scks - structure contains active fds and count
```

Comment:
Could you elaborate more on the purpose of introducing this structure. For example you can say something like if this structure is populated then we can only process the set of sockets present only in this set as a priority, else if this parameter is NULL then we process all sockets.

Reply:
done